src/libstd/ffi/c_str.rs

```diff
@@ -1175,9 +1175,9 @@ impl CStr {
     /// If the contents of the `CStr` are valid UTF-8 data, this
     /// function will return a [`Cow`]`::`[`Borrowed`]`(`[`&str`]`)`
     /// with the the corresponding [`&str`] slice. Otherwise, it will
-    /// replace any invalid UTF-8 sequences with `U+FFFD REPLACEMENT
-    /// CHARACTER` and return a [`Cow`]`::`[`Owned`]`(`[`String`]`)`
-    /// with the result.
+    /// replace any invalid UTF-8 sequences with
+    /// [`U+FFFD REPLACEMENT CHARACTER`][U+FFFD] and return a
```

Comment:
Looks like we're a bit inconsistent between "`U+FFFD REPLACEMENT CHARACTER`" and "U+FFFD REPLACEMENT CHARACTER".

Reply:
What's your opinion? I don't feel strongly

Reply:
I don't really feel strongly either.

Reply:
Made consistent in the latest force push